src/test/java/org/fungover/storm/config/configStub.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
If we have a config file in resources folder for production we can have a file with same name in resource folder under test. Then when running tests it will automatically use that file.

Reply:
Then I'll change it from configStub.json to config.json